src/renderer/components/publication/TagManager.tsx

```diff
@@ -68,15 +70,24 @@ export class TagManager extends React.Component<Props, TagManagerState> {
                     )}
                 </ul>}
                 {this.props.canModifyTag &&
-                    <form onSubmit={this.addTag} id={styles.flux_search}>
+                    <form id={styles.flux_search}>
                         <input
                             type="text"
                             className={styles.tag_inputs}
                             title={ __("catalog.addTags")}
                             placeholder={ __("catalog.addTags")}
                             onChange={this.handleChangeName}
                             value={this.state.nameNewTag}
+                            onKeyDown={(e) => {
+                                if (e.keyCode === 13) {
+                                    e.preventDefault();
+                                    return (false);
+                                }
+                            }}
                         />
+                        <button onClick={this.addTag}>
+                            <SVG svg={PlusIcon} title={__("catalog.confirmAddTag")}/>
+                        </button>
```

Comment:
The `+` button should be small, and inlined to the right of the text input (we will need to mirror this in order to support Right-To-Left languages, but this will be for a future PR with a global UI scope).

See screenshot:

![ThoriumAddButton](https://user-images.githubusercontent.com/625201/62067610-05601980-b22c-11e9-85d4-02b1363f6ee5.png)